samples/bookinfo/kube/egress-rule-google-apis.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: config.istio.io/v1alpha2
+kind: EgressRule
```

Comment:
Isn't this resource named `ExternalService` in apiVersion `v1alpha2`?

Reply:
@rshriram @frankbu There is some confusion with the versions. From what I see, all the routing and egress rules were `v1alpha2` already five months ago, e.g. https://github.com/istio/istio/blob/master/pilot/test/integration/testdata/rule-regex-route.yaml.tmpl.

In https://github.com/istio/istio/blob/master/pilot/test/integration/testdata all the route and egress rules are `v1alpha2`:
```
~/go/src/istio.io/istio/pilot/test/integration/testdata$ grep v1alpha2 * | wc -l
18
~/go/src/istio.io/istio/pilot/test/integration/testdata$ grep v1alpha1 * | wc -l
0
```

And also on istio.io, for example in https://istio.io/docs/tasks/traffic-management/request-timeouts.html.

Regarding  the question:
> Isn't this resource named ExternalService in apiVersion v1alpha2?

It is the resource named `ExternalService` in the next api version. Once it will be implemented by https://github.com/istio/istio/pull/2811, the tests for Egress Rules should be rewritten to use `ExternalService`.